theme_monglia/static/src/snippets/s_numbers/000.scss

```diff
@@ -1,8 +1,8 @@
 .s_numbers {
     .s_number {
-        font-size: 75px;
+        @include font-size(75px);
         @include media-breakpoint-down(md) {
-            font-size: 60px;
+            @include font-size(60px);
```

Comment:
Using the font-size mixin and doing "if > md, use X else use Y" probably is useless, no? Doing `@include font-size(75px);`, I guess you should have something like 60px in md automatically, no?

Reply:
I can even remove it because the .s_number has the display-4 class on it :+1: